src/test/java/com/virtuslab/archunit/UIThreadUnsafeMethodInvocationsTestSuite.java

```diff
@@ -139,6 +141,7 @@ public void check(JavaMethod method, ConditionEvents events) {
       "git4idea.branch.GitNewBranchDialog.showAndGetOptions()",
       "git4idea.branch.GitNewBranchOptions.getName()",
       "git4idea.branch.GitNewBranchOptions.shouldCheckout()",
+      "git4idea.config.GitConfigUtil.getBooleanValue(java.lang.String)",
```

Comment:
`getBooleanValue` is just a helper to convert string (typically retrieved via `GitConfigUtil.getValue`) to a `Boolean`, it doesn't really access git config